<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CArrangeOptionDlg</class>
 <widget class="QDialog" name="CArrangeOptionDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arranger Option</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>186</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="QLabel" name="labBlockHeight">
        <property name="text">
         <string>Block Height:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="txtBlockHeight"/>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="txtAreaHeight"/>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="txtBlockSpanVer"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labBlockWidth">
        <property name="text">
         <string>Block Width:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="labBlockSpanVer">
        <property name="text">
         <string>Block Span Ver:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labAreaHeight">
        <property name="text">
         <string>Max Area Height:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtBlockWidth"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txtBlockSpanHor"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labBlockSpanHor">
        <property name="text">
         <string>Block Span Hor:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labAreaWidth">
        <property name="text">
         <string>Max Area Width:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtAreaWidth"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labBlockMarginLeft">
        <property name="text">
         <string>Block Margin Left:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="txtBlockMarginLeft"/>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="labBlockMarginTop">
        <property name="text">
         <string>Block Margin Top:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="txtBlockMarginTop"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtAreaWidth</tabstop>
  <tabstop>txtAreaHeight</tabstop>
  <tabstop>txtBlockWidth</tabstop>
  <tabstop>txtBlockHeight</tabstop>
  <tabstop>txtBlockSpanHor</tabstop>
  <tabstop>txtBlockSpanVer</tabstop>
  <tabstop>txtBlockMarginLeft</tabstop>
  <tabstop>txtBlockMarginTop</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CArrangeOptionDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CArrangeOptionDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
